app.rb

```diff
@@ -33,17 +34,27 @@
   `/maestro toggle repeat` -- Toggles repeat playback mode.
 HELP_TEXT
 
-def spotify(args)
-  return [HELP_TEXT, true] if args == "help"
-  puts "# > spotify #{args}"
-  output = `./spotify.sh #{args}`
-  puts output
-  output = HELP_TEXT unless $?.success?
-  [output, $?.success?]
+# Find methods unique to Spotify class
+VALID_COMMANDS = (Spotify.methods - Object.methods).freeze
```

Comment:
Should this be `Spotify.public_methods`?

Reply:
With this PR, it won't make a difference, but for the sake of maintainability I'll change it.